.github/workflows/build.yml

```
@@ -151,7 +151,7 @@ jobs:
           key: ${{ hashFiles('.github/workflows/cache.yml') }}-${{ runner.os }}-${{ env.OS_ARCH }}-${{ hashFiles('tools/Cargo.lock') }}-${{ hashFiles('.github/workflows/build.yml') }}
           restore-keys: |
             ${{ hashFiles('.github/workflows/cache.yml') }}-${{ runner.os }}-${{ env.OS_ARCH }}-${{ hashFiles('tools/Cargo.lock') }}
-      - run: rustup default 1.66.1 && rustup component add rustfmt && rustup component add clippy
+      - run: rustup default 1.67.1 && rustup component add rustfmt
```

```suggestion
      - run: rustup component add rustfmt
```
I just realized, now that `clippy` runs from the SDK we actually don't have to pin at all!